### DISCUSSION TITLE
[1] First Blog Post

### DISCUSSION THREAD

**Discussion by GermanJesus-lul** [GermanJesus-lul/Rezeptbuch#2] [Announcements]
# Team Introduction
**Luca Grimm:** UI/UX-Design-/Development; CI/CD expert 
**Julius Göler:** Application logic developer
**Jaron Köhler:** API/Database developer
**Tom Jenrich:** API/Database developer

# Product Vision
A cookbook application designed for offline use, with features for sharing and discovering new recipes through add-ons that enhance your personal collection. Key functionalities include:
- Adjusting the number of servings
- Searching recipes by category
- Filtering recipes by ingredients, with the option to input on-hand ingredients and receive matching recipe suggestions

# Tech Stack
The application will be developed using .NET MAUI, with .NET handling the core application logic.

The choice of API technology is still under consideration. Our two API developers are open to suggestions and would welcome any recommendations.

As for the database technology, this will be determined during the elaboration phase. At that point, we will assess the best approach for storing recipes and select the most suitable solution.

# Scrum Setup
We use Jira as our Scrum management tool. Our development cycle consists of three-week sprints, with weekly meetings to stay on track. Each sprint begins with a planning session and includes a retrospective to review the previous sprint's progress and improvements.

# Sprint Backlog (Sprint #1)
In the first sprint, our focus will be on setting up the necessary software tools and refining the product vision to ensure a clear direction moving forward.
![image](https://github.com/user-attachments/assets/5718721e-59ef-4f32-b21d-359f2f57ba2b)

![image](https://github.com/user-attachments/assets/e001293e-4cb6-429e-a7c5-8d875be66c9e)

**Comment by philipsagawe** [comment]
Hallo zusammen,
Das Team-Setup klingt sehr vielversprechend und deckt alle wichtigen Bereiche der App-Entwicklung ab. Besonders beeindruckt hat mich eure Produktvision – eine Kochbuch-App mit Offline-Funktionalität und der Möglichkeit, Rezepte durch Add-ons zu erweitern, bietet echten Mehrwert für Nutzer. Auch die Idee, Rezepte nach Zutaten zu filtern, die bereits verfügbar sind, ist extrem nützlich!
Die Wahl von .NET MAUI für die plattformübergreifende Entwicklung finde ich spannend. Habt ihr schon bestimmte Frameworks oder Tools in Erwägung gezogen, die eure API-Entwicklung unterstützen könnten? Es wäre vielleicht sinnvoll, Technologien wie GraphQL oder REST zu vergleichen, um zu sehen, welche am besten zu euren Anforderungen passt. 
Auch eure Scrum-Organisation mit dreiwöchigen Sprints klingt strukturiert und gut geplant. Ich bin gespannt, wie sich das Projekt entwickelt, und wünsche euch viel Erfolg beim Kochen!
LG Philip
